doc/Manual.md

```diff
@@ -952,7 +963,8 @@ CABRILLO=cwo
 is in the rules file for the CW Open contest. All QSOs will then be formatted
 as in the following example:  
 
-QSO: 28000 CW 2012-09-23 0815 DL1JBE        0001 Tom       HB9XXX        0023 Joe QSO: .....  
+QSO: 28000 CW 2012-09-23 0815 DL1JBE        0001 Tom       HB9XXX        0023
```

Comment:
The second ellipsized QSO line can be dropped and the spaces in the first one reduced so that it fits one line.

Reply:
Fixed.